StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -70,17 +78,17 @@ final class UpdatePaymentMethodViewController: UIViewController {
     }()
 
     private lazy var updateButton: ConfirmButton = {
-        let button = ConfirmButton(state: .disabled, callToAction: .custom(title: .Localized.save), appearance: viewModel.appearance, didTap: {  [weak self] in
-            switch self?.viewModel.paymentMethod.type {
-            case .card:
+        let button = ConfirmButton(state: .disabled, callToAction: .custom(title: .Localized.save), appearance: viewModel.appearance, didTap: {  [self] in
```

Comment:
Why'd we remove `weak`? This was there to prevent a retain cycle.
https://medium.com/@mahigarg/retain-cycle-in-swift-45e5cbec3af4